src/q2g-ext-altStateDirective.ts

```diff
@@ -554,7 +554,6 @@ class AltStateController {
      */
     private createWarningMessage(): string {
         let msg: string = "";
-        setTimeout(500);
```

Comment:
@thomashaenig can u have a look please ? This is syntacical incorrect since this should be setTimeout(() => void, timeToWait), i removed this now since this breaks the build and extension. What was the expected behavior of this one ?